### DISCUSSION TITLE
Auto Design data export into Excel format

### DISCUSSION THREAD

**Discussion by SaiBhyri** [Telecominfraproject/oopt-gnpy#482] [Q&A] [answered]
Hi, 

Is it possible to export the auto design data into excel?
I see that there is a function to export the auto design into JSON.
I tried the same function but changed the extension to .xlsx. It generated the file but says corrupted on opening.

Thanks,
Sai

**Comment by EstherLerouzic** [comment, marked as answer]
Dear Sai,

autodesign export is only supported in JSON format. There is no plan to export in excel format.

regards
Esther

**Reply by SaiBhyri** [reply]
Thanks Esther.

Regards,
Sai
